frontend/src/components/editor/inputs/Inputs.styles.ts

```diff
@@ -18,7 +18,7 @@ export const button = cva(
         yellow: "mo-button yellow",
         // for actions that will change state but are not destructive
         "hint-green": "mo-button hint-green",
-        disabled: "mo-button disabled",
+        disabled: "mo-button disabled active:shadow-xsSolid",
```

Comment:
This is a small detail, but it prevents the user from visually depressing the button when it is disabled.